packages/blade/src/components/Radio/RadioIcon.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import { useTheme } from '~components/BladeProvider';
+import Svg from '~components/Icons/_Svg';
+import { getIn } from '~utils';
+import { CheckboxIconWrapper } from '~components/Checkbox/CheckboxIcon/CheckboxIconWrapper';
```

Comment:
Lets we extract this to a common place now? `InputIconWrapper`?

Reply:
Yep already done in #607